pkg/common/aws/ec2.go

```diff
@@ -79,3 +85,49 @@ func (CcsAwsSession *ccsAwsSession) ReleaseElasticIPs(dryrun bool, sendSummary b
 
 	return nil
 }
+
+// TerminateEC2Instances finds EC2 instances with tag Name "osde2e*",
+// older than given duration, then terminates these EC2 instances.
+// Ignores EC2 instances with tag Name "osde2e-proxy*".
+func (CcsAwsSession *ccsAwsSession) TerminateEC2Instances(olderthan time.Duration, dryrun bool, sendSummary bool,
+	terminatedCounter *int, failedCounter *int, errorBuilder *strings.Builder,
+) error {
```

Comment:
Please format this so it is one param per line like so or the entire definition on a single line. The rest of the file needs cleaner formatting in a later PR.

```suggestion
func (CcsAwsSession *ccsAwsSession) TerminateEC2Instances(
        olderthan time.Duration,
        dryrun bool,
        sendSummary bool,
	terminatedCounter *int,
	failedCounter *int,
	errorBuilder *strings.Builder,
) error {
```